services/core/ThresholdDetectionAgent/thresholddetection/agent.py

```diff
@@ -66,20 +66,19 @@
 
 utils.setup_logging()
 _log = logging.getLogger(__name__)
-__version__ = '3.6'
+__version__ = '3.7'
 
 
 def thresholddetection_agent(config_path, **kwargs):
     """Load configuration for ThresholdDetectionAgent
 
-    :param config_path: Path to a configuration file.
+    :param config_path: Path to a configuration file. Ignored.
```

Comment:
Shouldn't ignore, we should instead put them as defaults. and use the set_default function of the config store.  See actuator for example